core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/V1WorkspaceMigrationMonitorSpec.scala

```diff
@@ -39,4 +55,94 @@ class V1WorkspaceMigrationMonitorSpec
       }
     }
   }
+
+  // use an existing test project (broad-dsde-dev)
+  "temp Integration Test" should "for realsies create a new bucket in the same region" ignore {
+    val sourceProject = "general-dev-billing-account"
+    val sourceBucket = "az-leotest"
+    val destProject = "terra-dev-7af423b8"
+    val config = ConfigFactory.load()
+    val gcsConfig = config.getConfig("gcs")
+    val serviceProject = GoogleProject(sourceProject)
+    val pathToCredentialJson = "config/rawls-account.json"
+    val v1WorkspaceCopy = minimalTestData.v1Workspace.copy(namespace = sourceProject, googleProjectId = GoogleProjectId(sourceProject), bucketName = sourceBucket)
+
+    withMinimalTestDatabase { _ =>
+      runAndWait {
+        DBIO.seq(
+          workspaceQuery.createOrUpdate(v1WorkspaceCopy),
+          V1WorkspaceMigrationMonitor.schedule(v1WorkspaceCopy)
+        )
+      }
+      val attempt = runAndWait(migrations.filter(_.workspaceId === v1WorkspaceCopy.workspaceIdAsUUID).result).head
+      val writeAction = GoogleStorageService.resource[IO](pathToCredentialJson, None, Option(serviceProject)).use { googleStorageService =>
+        for {
+          res <- V1WorkspaceMigrationMonitor.createTempBucket(attempt, v1WorkspaceCopy, GoogleProject(destProject), googleStorageService)
+          (bucketName, writeAction) = res
+          loadedBucket <- googleStorageService.getBucket(GoogleProject(destProject), bucketName)
+          _ <- googleStorageService.deleteBucket(GoogleProject(destProject), bucketName).compile.drain
+        } yield {
+          loadedBucket shouldBe defined
+          writeAction
+        }
+      }.unsafeRunSync
+      runAndWait(writeAction)
+    }
+  }
+
+  "createTempBucket" should "create a new bucket in the same region" in {
+    val sourceProject = "something"
+    val sourceBucketName = "something_bucket"
+    val destProject = "something_else"
+    val v1Workspace = minimalTestData.v1Workspace.copy(
+      namespace = sourceProject,
+      googleProjectId = GoogleProjectId(sourceProject),
+      bucketName = sourceBucketName
+    )
+
+    val googleStorageService = mock[GoogleStorageService[IO]](RETURNS_SMART_NULLS)
+    val sourceBucket = mock[Bucket](RETURNS_SMART_NULLS)
```

Comment:
I've been unable to fix the mocking in this file. 

What I'd _like_ to do is just mock the `GoogleStorageService` and just return a `Bucket` object of my own creation for the `when` call down on l.109. However, I've been unable to instantiate a `Bucket` object as it doesn't appear to be publicly exposed by Google. I can create a `BucketInfo`, but not a `Bucket` without calling Google which defeats the purpose. Maybe there's a way to do this, but I haven't figured it out.

I tried going down this path as well and mocking out the `Bucket` class itself, but I'm getting an error from Mockito that says `Mockito cannot mock this class: class com.google.cloud.storage.Bucket.`. SO tells me that this is related to incompatibility between the Java and Mockito versions, but our version of Mockito is relatively recent and I don't feel like dealing with that.